web/test-utils/src/jsMain/kotlin/org/jetbrains/compose/web/testutils/TestUtils.kt

```diff
@@ -32,13 +21,18 @@ import kotlin.time.DurationUnit
 import kotlin.time.ExperimentalTime
 import kotlin.time.toDuration
 
+interface TestComposition {
```

Comment:
I don't see any problem with adding this interface. Just want to understand its purpose.
What problem does it solve?